### ISSUE TITLE
Need a better single line explainanation 

### ISSUE THREAD

**Issue by lakshit1001** [yeoman/yeoman.github.io#742]
TLDR 
i never used yeoman before coz this line confused me
"The web's scaffolding tool for modern webapps"
 and i have tried it and loved it 

FULL story
I am a web developer and heard about yeoman loads of time. I have even visited the website 500 times before today. I always came here wanted to get started but could never do the same. As I never understood what Yeoman actually is because of this first line 

"he web's scaffolding tool for modern webapps"

I googled to understand this and this confused me even more. 

![image](https://user-images.githubusercontent.com/6951276/30518002-fd9fa8ce-9b8e-11e7-8558-de84ce10be43.png)

I am not saying that the line is not great but just pointing out the fact that a lot of developers do not have English as their native or first language. It would be much better if we could have a simpler and clearer first line. 
:+1:

**Comment by SBoudrias** [comment]
We've been talking about alternative in #568

**Comment by mischah** [comment]
Thanks for the feedback @lakshit1001 👌
